source/content/guides/drupal/drupal-hosted-createempty-md/01-introduction.md

```diff
@@ -28,6 +28,14 @@ This guide will show you how to migrate a site that meets the following criteria
 
 <Partial file="drupal/commit-history.md" />
 
+This doc uses the following aliases:
+
+- **Alias:** `D8_SITE`
+  - **Site Name:** `best-drupal8-site-ever`
+- **Alias:** `DRUPAL_SITE`
+  - **Site Name:** `best-drupal-site-ever`
```

Comment:
Neither of these defines the `SITE` alias that is used on Lines 47 and 52 (39 and 44 in the original version of the file). For those commands to work, we need something like

```bash
SITE=anita-drupal
```

or

```bash
export SITE=anita-drupal
```

Reply:
Updated thanks to your feedback @benjifisher !